README.md
# Laravel Ecuador Locations

[![Latest Version on Packagist](https://img.shields.io/packagist/v/luilliarcec/laravel-ecuador-locations.svg)](https://packagist.org/packages/luilliarcec/laravel-ecuador-locations)
[![Total Downloads](https://img.shields.io/packagist/dt/luilliarcec/laravel-ecuador-locations)](https://packagist.org/packages/luilliarcec/laravel-ecuador-locations)
[![GitHub license](https://img.shields.io/github/license/luilliarcec/laravel-ecuador-locations.svg)](https://github.com/luilliarcec/laravel-ecuador-locations/blob/master/LICENSE.md)

## Installation

You can install the package via composer:

```bash
composer require luilliarcec/laravel-ecuador-locations
```

Now publish the migration file into your app's migrations directory, by running the following command:

```bash
php artisan vendor:publish --provider="Luilliarcec\EcuadorLocations\EcuadorLocationsServiceProvider"
```

```bash
php artisan migrate
```

### Usage

```php
<?php

namespace Database\Seeders;

use Illuminate\Database\Seeder;
use Luilliarcec\EcuadorLocations\Seeders\CantonSeeder;
use Luilliarcec\EcuadorLocations\Seeders\ProvinciaSeeder;
use Luilliarcec\EcuadorLocations\Models\Canton;
use Luilliarcec\EcuadorLocations\Models\Provincia;

class DatabaseSeeder extends Seeder
{
    public function run()
    {
        if (Provincia::doesntExist()) {
            $this->call(ProvinciaSeeder::class);
        }

        if (Canton::doesntExist()) {
            $this->call(CantonSeeder::class);
        }

        // ...
    }
}
```

```bash
php artisan db:seed
```

### Testing

``` bash
composer test
```

### Changelog

Please see [CHANGELOG](CHANGELOG.md) for more information what has changed recently.

## Contributing

Please see [CONTRIBUTING](CONTRIBUTING.md) for details.

### Security

If you discover any security related issues, please email luilliarcec@gmail.com instead of using the issue tracker.

## Credits

- [Luis Andrés Arce C.](https://github.com/luilliarcec)
- [All Contributors](../../contributors)

## License

The MIT License (MIT). Please see [License File](LICENSE.md) for more information.


CHANGELOG.md
# Changelog

All notable changes to `laravel-ecuador-locations` will be documented in this file

## 1.0.0 - 2021-10-13
- initial release


src/Seeders/ProvinciaSeeder.php
<?php

namespace Luilliarcec\EcuadorLocations\Seeders;

use Illuminate\Database\Seeder;
use Luilliarcec\EcuadorLocations\Models\Provincia;

class ProvinciaSeeder extends Seeder
{
    public function run()
    {
        Provincia::insert([
            ['id' => 1, 'code' => '01', 'name' => 'AZUAY'],
            ['id' => 2, 'code' => '02', 'name' => 'BOLÍVAR'],
            ['id' => 3, 'code' => '03', 'name' => 'CAÑAR'],
            ['id' => 4, 'code' => '04', 'name' => 'CARCHI'],
            ['id' => 5, 'code' => '05', 'name' => 'COTOPAXI'],
            ['id' => 6, 'code' => '06', 'name' => 'CHIMBORAZO'],
            ['id' => 7, 'code' => '07', 'name' => 'EL ORO'],
            ['id' => 8, 'code' => '08', 'name' => 'ESMERALDAS'],
            ['id' => 9, 'code' => '09', 'name' => 'GUAYAS'],
            ['id' => 10, 'code' => '10', 'name' => 'IMBABURA'],
            ['id' => 11, 'code' => '11', 'name' => 'LOJA'],
            ['id' => 12, 'code' => '12', 'name' => 'LOS RÍOS'],
            ['id' => 13, 'code' => '13', 'name' => 'MANABÍ'],
            ['id' => 14, 'code' => '14', 'name' => 'MORONA SANTIAGO'],
            ['id' => 15, 'code' => '15', 'name' => 'NAPO'],
            ['id' => 16, 'code' => '16', 'name' => 'PASTAZA'],
            ['id' => 17, 'code' => '17', 'name' => 'PICHINCHA'],
            ['id' => 18, 'code' => '18', 'name' => 'TUNGURAHUA'],
            ['id' => 19, 'code' => '19', 'name' => 'ZAMORA CHINCHIPE'],
            ['id' => 20, 'code' => '20', 'name' => 'GALÁPAGOS'],
            ['id' => 21, 'code' => '21', 'name' => 'SUCUMBÍOS'],
            ['id' => 22, 'code' => '22', 'name' => 'ORELLANA'],
            ['id' => 23, 'code' => '23', 'name' => 'SANTO DOMINGO DE LOS TSÁCHILAS'],
            ['id' => 24, 'code' => '24', 'name' => 'SANTA ELENA'],
        ]);
    }
}


src/Seeders/CantonSeeder.php
<?php

namespace Luilliarcec\EcuadorLocations\Seeders;

use Illuminate\Database\Seeder;
use Luilliarcec\EcuadorLocations\Models\Canton;

class CantonSeeder extends Seeder
{
    public function run()
    {
        Canton::insert([
            ['id' => 1, 'provincia_id' => 1, 'code' => '0101', 'name' => 'CUENCA'],
            ['id' => 2, 'provincia_id' => 1, 'code' => '0102', 'name' => 'GIRÓN'],
            ['id' => 3, 'provincia_id' => 1, 'code' => '0103', 'name' => 'GUALACEO'],
            ['id' => 4, 'provincia_id' => 1, 'code' => '0104', 'name' => 'NABÓN'],
            ['id' => 5, 'provincia_id' => 1, 'code' => '0105', 'name' => 'PAUTE'],
            ['id' => 6, 'provincia_id' => 1, 'code' => '0106', 'name' => 'PUCARÁ'],
            ['id' => 7, 'provincia_id' => 1, 'code' => '0107', 'name' => 'SAN FERNANDO'],
            ['id' => 8, 'provincia_id' => 1, 'code' => '0108', 'name' => 'SANTA ISABEL'],
            ['id' => 9, 'provincia_id' => 1, 'code' => '0109', 'name' => 'SÍGSIG'],
            ['id' => 10, 'provincia_id' => 1, 'code' => '0110', 'name' => 'OÑA'],
            ['id' => 11, 'provincia_id' => 1, 'code' => '0111', 'name' => 'CHORDELEG'],
            ['id' => 12, 'provincia_id' => 1, 'code' => '0112', 'name' => 'EL PAN'],
            ['id' => 13, 'provincia_id' => 1, 'code' => '0113', 'name' => 'SEVILLA DE ORO'],
            ['id' => 14, 'provincia_id' => 1, 'code' => '0114', 'name' => 'GUACHAPALA'],
            ['id' => 15, 'provincia_id' => 1, 'code' => '0115', 'name' => 'CAMILO PONCE ENRÍQUEZ'],
            ['id' => 16, 'provincia_id' => 2, 'code' => '0201', 'name' => 'GUARANDA'],
            ['id' => 17, 'provincia_id' => 2, 'code' => '0202', 'name' => 'CHILLANES'],
            ['id' => 18, 'provincia_id' => 2, 'code' => '0203', 'name' => 'CHIMBO'],
            ['id' => 19, 'provincia_id' => 2, 'code' => '0204', 'name' => 'ECHEANDÍA'],
            ['id' => 20, 'provincia_id' => 2, 'code' => '0205', 'name' => 'SAN MIGUEL'],
            ['id' => 21, 'provincia_id' => 2, 'code' => '0206', 'name' => 'CALUMA'],
            ['id' => 22, 'provincia_id' => 2, 'code' => '0207', 'name' => 'LAS NAVES'],
            ['id' => 23, 'provincia_id' => 3, 'code' => '0301', 'name' => 'AZOGUES'],
            ['id' => 24, 'provincia_id' => 3, 'code' => '0302', 'name' => 'BIBLIÁN'],
            ['id' => 25, 'provincia_id' => 3, 'code' => '0303', 'name' => 'CAÑAR'],
            ['id' => 26, 'provincia_id' => 3, 'code' => '0304', 'name' => 'LA TRONCAL'],
            ['id' => 27, 'provincia_id' => 3, 'code' => '0305', 'name' => 'EL TAMBO'],
            ['id' => 28, 'provincia_id' => 3, 'code' => '0306', 'name' => 'DÉLEG'],
            ['id' => 29, 'provincia_id' => 3, 'code' => '0307', 'name' => 'SUSCAL'],
            ['id' => 30, 'provincia_id' => 4, 'code' => '0401', 'name' => 'TULCÁN'],
            ['id' => 31, 'provincia_id' => 4, 'code' => '0402', 'name' => 'BOLÍVAR'],
            ['id' => 32, 'provincia_id' => 4, 'code' => '0403', 'name' => 'ESPEJO'],
            ['id' => 33, 'provincia_id' => 4, 'code' => '0404', 'name' => 'MIRA'],
            ['id' => 34, 'provincia_id' => 4, 'code' => '0405', 'name' => 'MONTÚFAR'],
            ['id' => 35, 'provincia_id' => 4, 'code' => '0406', 'name' => 'SAN PEDRO DE HUACA'],
            ['id' => 36, 'provincia_id' => 5, 'code' => '0501', 'name' => 'LATACUNGA'],
            ['id' => 37, 'provincia_id' => 5, 'code' => '0502', 'name' => 'LA MANÁ'],
            ['id' => 38, 'provincia_id' => 5, 'code' => '0503', 'name' => 'PANGUA'],
            ['id' => 39, 'provincia_id' => 5, 'code' => '0504', 'name' => 'PUJILÍ'],
            ['id' => 40, 'provincia_id' => 5, 'code' => '0505', 'name' => 'SALCEDO'],
            ['id' => 41, 'provincia_id' => 5, 'code' => '0506', 'name' => 'SAQUISILÍ'],
            ['id' => 42, 'provincia_id' => 5, 'code' => '0507', 'name' => 'SIGCHOS'],
            ['id' => 43, 'provincia_id' => 6, 'code' => '0601', 'name' => 'RIOBAMBA'],
            ['id' => 44, 'provincia_id' => 6, 'code' => '0602', 'name' => 'ALAUSÍ'],
            ['id' => 45, 'provincia_id' => 6, 'code' => '0603', 'name' => 'COLTA'],
            ['id' => 46, 'provincia_id' => 6, 'code' => '0604', 'name' => 'CHAMBO'],
            ['id' => 47, 'provincia_id' => 6, 'code' => '0605', 'name' => 'CHUNCHI'],
            ['id' => 48, 'provincia_id' => 6, 'code' => '0606', 'name' => 'GUAMOTE'],
            ['id' => 49, 'provincia_id' => 6, 'code' => '0607', 'name' => 'GUANO'],
            ['id' => 50, 'provincia_id' => 6, 'code' => '0608', 'name' => 'PALLATANGA'],
            ['id' => 51, 'provincia_id' => 6, 'code' => '0609', 'name' => 'PENIPE'],
            ['id' => 52, 'provincia_id' => 6, 'code' => '0610', 'name' => 'CUMANDÁ'],
            ['id' => 53, 'provincia_id' => 7, 'code' => '0701', 'name' => 'MACHALA'],
            ['id' => 54, 'provincia_id' => 7, 'code' => '0702', 'name' => 'ARENILLAS'],
            ['id' => 55, 'provincia_id' => 7, 'code' => '0703', 'name' => 'ATAHUALPA'],
            ['id' => 56, 'provincia_id' => 7, 'code' => '0704', 'name' => 'BALSAS'],
            ['id' => 57, 'provincia_id' => 7, 'code' => '0705', 'name' => 'CHILLA'],
            ['id' => 58, 'provincia_id' => 7, 'code' => '0706', 'name' => 'EL GUABO'],
            ['id' => 59, 'provincia_id' => 7, 'code' => '0707', 'name' => 'HUAQUILLAS'],
            ['id' => 60, 'provincia_id' => 7, 'code' => '0708', 'name' => 'MARCABELÍ'],
            ['id' => 61, 'provincia_id' => 7, 'code' => '0709', 'name' => 'PASAJE'],
            ['id' => 62, 'provincia_id' => 7, 'code' => '0710', 'name' => 'PIÑAS'],
            ['id' => 63, 'provincia_id' => 7, 'code' => '0711', 'name' => 'PORTOVELO'],
            ['id' => 64, 'provincia_id' => 7, 'code' => '0712', 'name' => 'SANTA ROSA'],
            ['id' => 65, 'provincia_id' => 7, 'code' => '0713', 'name' => 'ZARUMA'],
            ['id' => 66, 'provincia_id' => 7, 'code' => '0714', 'name' => 'LAS LAJAS'],
            ['id' => 67, 'provincia_id' => 8, 'code' => '0801', 'name' => 'ESMERALDAS'],
            ['id' => 68, 'provincia_id' => 8, 'code' => '0802', 'name' => 'ELOY ALFARO'],
            ['id' => 69, 'provincia_id' => 8, 'code' => '0803', 'name' => 'MUISNE'],
            ['id' => 70, 'provincia_id' => 8, 'code' => '0804', 'name' => 'QUININDÉ'],
            ['id' => 71, 'provincia_id' => 8, 'code' => '0805', 'name' => 'SAN LORENZO'],
            ['id' => 72, 'provincia_id' => 8, 'code' => '0806', 'name' => 'ATACAMES'],
            ['id' => 73, 'provincia_id' => 8, 'code' => '0807', 'name' => 'RIOVERDE'],
            ['id' => 74, 'provincia_id' => 9, 'code' => '0901', 'name' => 'GUAYAQUIL'],
            ['id' => 75, 'provincia_id' => 9, 'code' => '0902', 'name' => 'ALFREDO BAQUERIZO MORENO (JUJÁN)'],
            ['id' => 76, 'provincia_id' => 9, 'code' => '0903', 'name' => 'BALAO'],
            ['id' => 77, 'provincia_id' => 9, 'code' => '0904', 'name' => 'BALZAR'],
            ['id' => 78, 'provincia_id' => 9, 'code' => '0905', 'name' => 'COLIMES'],
            ['id' => 79, 'provincia_id' => 9, 'code' => '0906', 'name' => 'DAULE'],
            ['id' => 80, 'provincia_id' => 9, 'code' => '0907', 'name' => 'DURÁN'],
            ['id' => 81, 'provincia_id' => 9, 'code' => '0908', 'name' => 'EL EMPALME'],
            ['id' => 82, 'provincia_id' => 9, 'code' => '0909', 'name' => 'EL TRIUNFO'],
            ['id' => 83, 'provincia_id' => 9, 'code' => '0910', 'name' => 'MILAGRO'],
            ['id' => 84, 'provincia_id' => 9, 'code' => '0911', 'name' => 'NARANJAL'],
            ['id' => 85, 'provincia_id' => 9, 'code' => '0912', 'name' => 'NARANJITO'],
            ['id' => 86, 'provincia_id' => 9, 'code' => '0913', 'name' => 'PALESTINA'],
            ['id' => 87, 'provincia_id' => 9, 'code' => '0914', 'name' => 'PEDRO CARBO'],
            ['id' => 88, 'provincia_id' => 9, 'code' => '0916', 'name' => 'SAMBORONDÓN'],
            ['id' => 89, 'provincia_id' => 9, 'code' => '0918', 'name' => 'SANTA LUCÍA'],
            ['id' => 90, 'provincia_id' => 9, 'code' => '0919', 'name' => 'SALITRE'],
            ['id' => 91, 'provincia_id' => 9, 'code' => '0920', 'name' => 'SAN JACINTO DE YAGUACHI'],
            ['id' => 92, 'provincia_id' => 9, 'code' => '0921', 'name' => 'PLAYAS'],
            ['id' => 93, 'provincia_id' => 9, 'code' => '0922', 'name' => 'SIMÓN BOLÍVAR'],
            ['id' => 94, 'provincia_id' => 9, 'code' => '0923', 'name' => 'CORONEL MARCELINO MARIDUEÑA'],
            ['id' => 95, 'provincia_id' => 9, 'code' => '0924', 'name' => 'LOMAS DE SARGENTILLO'],
            ['id' => 96, 'provincia_id' => 9, 'code' => '0925', 'name' => 'NOBOL'],
            ['id' => 97, 'provincia_id' => 9, 'code' => '0927', 'name' => 'GENERAL ANTONIO ELIZALDE'],
            ['id' => 98, 'provincia_id' => 9, 'code' => '0928', 'name' => 'ISIDRO AYORA'],
            ['id' => 99, 'provincia_id' => 10, 'code' => '1001', 'name' => 'IBARRA'],
            ['id' => 100, 'provincia_id' => 10, 'code' => '1002', 'name' => 'ANTONIO ANTE'],
            ['id' => 101, 'provincia_id' => 10, 'code' => '1003', 'name' => 'COTACACHI'],
            ['id' => 102, 'provincia_id' => 10, 'code' => '1004', 'name' => 'OTAVALO'],
            ['id' => 103, 'provincia_id' => 10, 'code' => '1005', 'name' => 'PIMAMPIRO'],
            ['id' => 104, 'provincia_id' => 10, 'code' => '1006', 'name' => 'SAN MIGUEL DE URCUQUÍ'],
            ['id' => 105, 'provincia_id' => 11, 'code' => '1101', 'name' => 'LOJA'],
            ['id' => 106, 'provincia_id' => 11, 'code' => '1102', 'name' => 'CALVAS'],
            ['id' => 107, 'provincia_id' => 11, 'code' => '1103', 'name' => 'CATAMAYO'],
            ['id' => 108, 'provincia_id' => 11, 'code' => '1104', 'name' => 'CELICA'],
            ['id' => 109, 'provincia_id' => 11, 'code' => '1105', 'name' => 'CHAGUARPAMBA'],
            ['id' => 110, 'provincia_id' => 11, 'code' => '1106', 'name' => 'ESPÍNDOLA'],
            ['id' => 111, 'provincia_id' => 11, 'code' => '1107', 'name' => 'GONZANAMÁ'],
            ['id' => 112, 'provincia_id' => 11, 'code' => '1108', 'name' => 'MACARÁ'],
            ['id' => 113, 'provincia_id' => 11, 'code' => '1109', 'name' => 'PALTAS'],
            ['id' => 114, 'provincia_id' => 11, 'code' => '1110', 'name' => 'PUYANGO'],
            ['id' => 115, 'provincia_id' => 11, 'code' => '1111', 'name' => 'SARAGURO'],
            ['id' => 116, 'provincia_id' => 11, 'code' => '1112', 'name' => 'SOZORANGA'],
            ['id' => 117, 'provincia_id' => 11, 'code' => '1113', 'name' => 'ZAPOTILLO'],
            ['id' => 118, 'provincia_id' => 11, 'code' => '1114', 'name' => 'PINDAL'],
            ['id' => 119, 'provincia_id' => 11, 'code' => '1115', 'name' => 'QUILANGA'],
            ['id' => 120, 'provincia_id' => 11, 'code' => '1116', 'name' => 'OLMEDO'],
            ['id' => 121, 'provincia_id' => 12, 'code' => '1201', 'name' => 'BABAHOYO'],
            ['id' => 122, 'provincia_id' => 12, 'code' => '1202', 'name' => 'BABA'],
            ['id' => 123, 'provincia_id' => 12, 'code' => '1203', 'name' => 'MONTALVO'],
            ['id' => 124, 'provincia_id' => 12, 'code' => '1204', 'name' => 'PUEBLOVIEJO'],
            ['id' => 125, 'provincia_id' => 12, 'code' => '1205', 'name' => 'QUEVEDO'],
            ['id' => 126, 'provincia_id' => 12, 'code' => '1206', 'name' => 'URDANETA'],
            ['id' => 127, 'provincia_id' => 12, 'code' => '1207', 'name' => 'VENTANAS'],
            ['id' => 128, 'provincia_id' => 12, 'code' => '1208', 'name' => 'VINCES'],
            ['id' => 129, 'provincia_id' => 12, 'code' => '1209', 'name' => 'PALENQUE'],
            ['id' => 130, 'provincia_id' => 12, 'code' => '1210', 'name' => 'BUENA FE'],
            ['id' => 131, 'provincia_id' => 12, 'code' => '1211', 'name' => 'VALENCIA'],
            ['id' => 132, 'provincia_id' => 12, 'code' => '1212', 'name' => 'MOCACHE'],
            ['id' => 133, 'provincia_id' => 12, 'code' => '1213', 'name' => 'QUINSALOMA'],
            ['id' => 134, 'provincia_id' => 13, 'code' => '1301', 'name' => 'PORTOVIEJO'],
            ['id' => 135, 'provincia_id' => 13, 'code' => '1302', 'name' => 'BOLÍVAR'],
            ['id' => 136, 'provincia_id' => 13, 'code' => '1303', 'name' => 'CHONE'],
            ['id' => 137, 'provincia_id' => 13, 'code' => '1304', 'name' => 'EL CARMEN'],
            ['id' => 138, 'provincia_id' => 13, 'code' => '1305', 'name' => 'FLAVIO ALFARO'],
            ['id' => 139, 'provincia_id' => 13, 'code' => '1306', 'name' => 'JIPIJAPA'],
            ['id' => 140, 'provincia_id' => 13, 'code' => '1307', 'name' => 'JUNÍN'],
            ['id' => 141, 'provincia_id' => 13, 'code' => '1308', 'name' => 'MANTA'],
            ['id' => 142, 'provincia_id' => 13, 'code' => '1309', 'name' => 'MONTECRISTI'],
            ['id' => 143, 'provincia_id' => 13, 'code' => '1310', 'name' => 'PAJÁN'],
            ['id' => 144, 'provincia_id' => 13, 'code' => '1311', 'name' => 'PICHINCHA'],
            ['id' => 145, 'provincia_id' => 13, 'code' => '1312', 'name' => 'ROCAFUERTE'],
            ['id' => 146, 'provincia_id' => 13, 'code' => '1313', 'name' => 'SANTA ANA'],
            ['id' => 147, 'provincia_id' => 13, 'code' => '1314', 'name' => 'SUCRE'],
            ['id' => 148, 'provincia_id' => 13, 'code' => '1315', 'name' => 'TOSAGUA'],
            ['id' => 149, 'provincia_id' => 13, 'code' => '1316', 'name' => '24 DE MAYO'],
            ['id' => 150, 'provincia_id' => 13, 'code' => '1317', 'name' => 'PEDERNALES'],
            ['id' => 151, 'provincia_id' => 13, 'code' => '1318', 'name' => 'OLMEDO'],
            ['id' => 152, 'provincia_id' => 13, 'code' => '1319', 'name' => 'PUERTO LÓPEZ'],
            ['id' => 153, 'provincia_id' => 13, 'code' => '1320', 'name' => 'JAMA'],
            ['id' => 154, 'provincia_id' => 13, 'code' => '1321', 'name' => 'JARAMIJÓ'],
            ['id' => 155, 'provincia_id' => 13, 'code' => '1322', 'name' => 'SAN VICENTE'],
            ['id' => 156, 'provincia_id' => 14, 'code' => '1401', 'name' => 'MORONA'],
            ['id' => 157, 'provincia_id' => 14, 'code' => '1402', 'name' => 'GUALAQUIZA'],
            ['id' => 158, 'provincia_id' => 14, 'code' => '1403', 'name' => 'LIMÓN INDANZA'],
            ['id' => 159, 'provincia_id' => 14, 'code' => '1404', 'name' => 'PALORA'],
            ['id' => 160, 'provincia_id' => 14, 'code' => '1405', 'name' => 'SANTIAGO'],
            ['id' => 161, 'provincia_id' => 14, 'code' => '1406', 'name' => 'SUCÚA'],
            ['id' => 162, 'provincia_id' => 14, 'code' => '1407', 'name' => 'HUAMBOYA'],
            ['id' => 163, 'provincia_id' => 14, 'code' => '1408', 'name' => 'SAN JUAN BOSCO'],
            ['id' => 164, 'provincia_id' => 14, 'code' => '1409', 'name' => 'TAISHA'],
            ['id' => 165, 'provincia_id' => 14, 'code' => '1410', 'name' => 'LOGROÑO'],
            ['id' => 166, 'provincia_id' => 14, 'code' => '1411', 'name' => 'PABLO SEXTO'],
            ['id' => 167, 'provincia_id' => 14, 'code' => '1412', 'name' => 'TIWINTZA'],
            ['id' => 168, 'provincia_id' => 15, 'code' => '1501', 'name' => 'TENA'],
            ['id' => 169, 'provincia_id' => 15, 'code' => '1503', 'name' => 'ARCHIDONA'],
            ['id' => 170, 'provincia_id' => 15, 'code' => '1504', 'name' => 'EL CHACO'],
            ['id' => 171, 'provincia_id' => 15, 'code' => '1507', 'name' => 'QUIJOS'],
            ['id' => 172, 'provincia_id' => 15, 'code' => '1509', 'name' => 'CARLOS JULIO AROSEMENA TOLA'],
            ['id' => 173, 'provincia_id' => 16, 'code' => '1601', 'name' => 'PASTAZA'],
            ['id' => 174, 'provincia_id' => 16, 'code' => '1602', 'name' => 'MERA'],
            ['id' => 175, 'provincia_id' => 16, 'code' => '1603', 'name' => 'SANTA CLARA'],
            ['id' => 176, 'provincia_id' => 16, 'code' => '1604', 'name' => 'ARAJUNO'],
            ['id' => 177, 'provincia_id' => 17, 'code' => '1701', 'name' => 'DISTRITO METROPOLITANO DE QUITO'],
            ['id' => 178, 'provincia_id' => 17, 'code' => '1702', 'name' => 'CAYAMBE'],
            ['id' => 179, 'provincia_id' => 17, 'code' => '1703', 'name' => 'MEJÍA'],
            ['id' => 180, 'provincia_id' => 17, 'code' => '1704', 'name' => 'PEDRO MONCAYO'],
            ['id' => 181, 'provincia_id' => 17, 'code' => '1705', 'name' => 'RUMIÑAHUI'],
            ['id' => 182, 'provincia_id' => 17, 'code' => '1707', 'name' => 'SAN MIGUEL DE LOS BANCOS'],
            ['id' => 183, 'provincia_id' => 17, 'code' => '1708', 'name' => 'PEDRO VICENTE MALDONADO'],
            ['id' => 184, 'provincia_id' => 17, 'code' => '1709', 'name' => 'PUERTO QUITO'],
            ['id' => 185, 'provincia_id' => 18, 'code' => '1801', 'name' => 'AMBATO'],
            ['id' => 186, 'provincia_id' => 18, 'code' => '1802', 'name' => 'BAÑOS DE AGUA SANTA'],
            ['id' => 187, 'provincia_id' => 18, 'code' => '1803', 'name' => 'CEVALLOS'],
            ['id' => 188, 'provincia_id' => 18, 'code' => '1804', 'name' => 'MOCHA'],
            ['id' => 189, 'provincia_id' => 18, 'code' => '1805', 'name' => 'PATATE'],
            ['id' => 190, 'provincia_id' => 18, 'code' => '1806', 'name' => 'QUERO'],
            ['id' => 191, 'provincia_id' => 18, 'code' => '1807', 'name' => 'SAN PEDRO DE PELILEO'],
            ['id' => 192, 'provincia_id' => 18, 'code' => '1808', 'name' => 'SANTIAGO DE PÍLLARO'],
            ['id' => 193, 'provincia_id' => 18, 'code' => '1809', 'name' => 'TISALEO'],
            ['id' => 194, 'provincia_id' => 19, 'code' => '1901', 'name' => 'ZAMORA'],
            ['id' => 195, 'provincia_id' => 19, 'code' => '1902', 'name' => 'CHINCHIPE'],
            ['id' => 196, 'provincia_id' => 19, 'code' => '1903', 'name' => 'NANGARITZA'],
            ['id' => 197, 'provincia_id' => 19, 'code' => '1904', 'name' => 'YACUAMBI'],
            ['id' => 198, 'provincia_id' => 19, 'code' => '1905', 'name' => 'YANTZAZA'],
            ['id' => 199, 'provincia_id' => 19, 'code' => '1906', 'name' => 'EL PANGUI'],
            ['id' => 200, 'provincia_id' => 19, 'code' => '1907', 'name' => 'CENTINELA DEL CÓNDOR'],
            ['id' => 201, 'provincia_id' => 19, 'code' => '1908', 'name' => 'PALANDA'],
            ['id' => 202, 'provincia_id' => 19, 'code' => '1909', 'name' => 'PAQUISHA'],
            ['id' => 203, 'provincia_id' => 20, 'code' => '2001', 'name' => 'SAN CRISTÓBAL'],
            ['id' => 204, 'provincia_id' => 20, 'code' => '2002', 'name' => 'ISABELA'],
            ['id' => 205, 'provincia_id' => 20, 'code' => '2003', 'name' => 'SANTA CRUZ'],
            ['id' => 206, 'provincia_id' => 21, 'code' => '2101', 'name' => 'LAGO AGRIO'],
            ['id' => 207, 'provincia_id' => 21, 'code' => '2102', 'name' => 'GONZALO PIZARRO'],
            ['id' => 208, 'provincia_id' => 21, 'code' => '2103', 'name' => 'PUTUMAYO'],
            ['id' => 209, 'provincia_id' => 21, 'code' => '2104', 'name' => 'SHUSHUFINDI'],
            ['id' => 210, 'provincia_id' => 21, 'code' => '2105', 'name' => 'SUCUMBÍOS'],
            ['id' => 211, 'provincia_id' => 21, 'code' => '2106', 'name' => 'CASCALES'],
            ['id' => 212, 'provincia_id' => 21, 'code' => '2107', 'name' => 'CUYABENO'],
            ['id' => 213, 'provincia_id' => 22, 'code' => '2201', 'name' => 'FRANCISCO DE ORELLANA'],
            ['id' => 214, 'provincia_id' => 22, 'code' => '2202', 'name' => 'AGUARICO'],
            ['id' => 215, 'provincia_id' => 22, 'code' => '2203', 'name' => 'LA JOYA DE LOS SACHAS'],
            ['id' => 216, 'provincia_id' => 22, 'code' => '2204', 'name' => 'LORETO'],
            ['id' => 217, 'provincia_id' => 23, 'code' => '2301', 'name' => 'SANTO DOMINGO'],
            ['id' => 218, 'provincia_id' => 23, 'code' => '2302', 'name' => 'LA CONCORDIA'],
            ['id' => 219, 'provincia_id' => 24, 'code' => '2401', 'name' => 'SANTA ELENA'],
            ['id' => 220, 'provincia_id' => 24, 'code' => '2402', 'name' => 'LA LIBERTAD'],
            ['id' => 221, 'provincia_id' => 24, 'code' => '2403', 'name' => 'SALINAS'],
        ]);
    }
}
